python/private/pypi/whl_library_targets.bzl

```diff
@@ -101,8 +102,16 @@ def whl_library_targets(
         srcs_exclude = [],
         tags = [],
         filegroups = {
-            DIST_INFO_LABEL: ["site-packages/*.dist-info/**"],
-            DATA_LABEL: ["data/**"],
+            EXTRACTED_WHEEL_FILES: dict(
+                include = ["**"],
```

Comment:
nit, in the repo phase we could parse the `RECORD` file and include those files instead? In the case where the whl is built from an sdist, this may include the extra `sdist` file under some configurations. Maybe we should exclude these things? The wheel patching thing has some logic to exclude it correctly.

Reply:
Ah, I forgot about sdists. And didn't realize they get extracted out into here (and then built into a wheel?).

Rather than parse RECORD, how about making the glob conditional? e.g. `if name.endswith(".whl"): glob-include **`

The RECORD is just a simple manifest of everything in the wheel file, right? i.e. `glob(**)` and RECORD should expand to the same set of files?

Reply:
name will always end with `.whl` because the sdist gets built into a wheel. Maybe a better way would be to remove the sdist or just add it to the excludes list.

Reply:
I like the idea of excluding it from the glob. I think I figured out how to pass the sdist filename (if one exists) along.

Reply:
I plumbed through the sdist filename from whl_library to also be excluded.